modules/widgets/facebook-likebox.php

```diff
@@ -42,6 +37,8 @@ function widget( $args, $instance ) {
 
 		$like_args = $this->normalize_facebook_args( $instance['like_args'] );
 
+		wp_enqueue_style( 'jetpack_facebook_likebox', plugins_url( 'facebook-likebox/style.css', __FILE__ ) );
```

Comment:
Yeah, on WordPress.com the file is concatenated so an "extra" file is preferred to the inline style. I wasn't aware of `maybe_inline_style`. I'll take a look.